scripts/generate-requirements

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Usage: ./scripts/generate-requirements [optional paths to additional requirements.txt]
```

Comment:
This argument parsing could be improved:

1. It could maybe take an output file / files for where to write stuff.
2. It doesn't use any of the fancy argument parsing stuff from the other scripts for things like help, etc.